test/test_cuda.py

```diff
@@ -4459,23 +4459,13 @@ def test_graph_optims(self, device, dtype, optim_info):
         all_optim_inputs = _get_optim_inputs_including_global_cliquey_kwargs(
             device, dtype, optim_info, skip=("differentiable",)
         )
-        has_betas = any(
-            "betas" in error_inp.optimizer_error_input.kwargs
-            for error_inp in optim_info.optim_error_inputs_func(
-                device="cpu", dtype=dtype
-            )
-        )
 
         steps_warmup = 3
         steps_train = 2
 
         for optim_input in all_optim_inputs:
             kwargs = optim_input.kwargs
-            if "lr" in kwargs:
-                del kwargs["lr"]
-            kwargs["lr"] = 0.1
-            if has_betas and optim_cls != torch.optim.Adamax:
-                kwargs["betas"] = (0.8, 0.7)
+            kwargs["lr"]=0.1
```

Comment:
wait this isn't necessary
```suggestion
```

Reply:
I have tried removing the lr reassignment, but I believe some of the kwargs for adam and adamw use tensor lrs which throws an error when capturable is False and foreach is True, should I just skip those tests where we use a tensor lr?

Reply:
Ah, I see, then yes. Having a comment about that will be good so the next person knows that's why we special case it.